docs/buildah-config.md

```diff
@@ -56,9 +56,13 @@ Note: this setting is not present in the OCIv1 image format, so it is discarded
 **--entrypoint** *entry*
```

Comment:
Good opportunity to update the --entrypoint syntax doco (provided by @TomSweeneyRedHat)

`--entrypoint "string"  | '["array_entry1", "array_entry2", ...]'`

Reply:
Would like to see the above somewhere in the man page and/or an example of each